src/types.ts

```diff
@@ -36,7 +36,7 @@ export interface SlotWithAvailability {
 }
 
 export interface AvailabilityDates {
-  [key: string]: SlotWithAvailability[] | undefined;
+  [key: string]: SlotWithAvailability[];
```

Comment:
why? i had `undefined` so that going `availabilityDates[date]` will give me `SlotWithAvailability | undefined`

Reply:
i reverted this